regression/test.pl

```diff
@@ -53,10 +53,7 @@ ($$$$$)
     }
   }
 
-  open my $FH, ">>$name/$output";
-  print $FH "EXIT=$exit_value\n";
-  print $FH "SIGNAL=$signal_num\n";
-  close $FH;
+  system("bash", "-c", "cd '$name' ; echo '\nEXIT=$exit_value\nSIGNAL=$signal_num\n' >> '$output'");
```

Comment:
This seems annoyingly messy and fragile - why has this not shown up before now - or is it related to the signal handling for sigalrm?

Reply:
It's related to sigalrm. That was the only robust way of getting the second output appended to the file without losing the leading newline.
In the end I'm not sure it's worth the hassle. Probably I should just remove these two tests.